src/update.js

```diff
@@ -0,0 +1,11 @@
+export function updateStatus(event) {
+  event.target.nextElementSibling.classList.toggle('completed');
+  const status = JSON.parse(localStorage.getItem('tasks'));
+  if (status[event.target.parentNode.parentNode.id].completed === true) {
+    status[event.target.parentNode.parentNode.id].completed = false;
+  } else {
+    status[event.target.parentNode.parentNode.id].completed = true;
+  }
+  localStorage.setItem('tasks', JSON.stringify(status));
+}
+export default updateStatus;
```

Comment:
- [ ] As you can see status still has not changed in the local storage. Kindly fix it. 

note: I refresh the storage.
![image](https://user-images.githubusercontent.com/38635158/185700780-bea46fb5-d9bb-47eb-ae1d-ec1a03dd2e06.png)